ui/app/components/flash-message.js

```diff
@@ -1,15 +1,11 @@
-import { computed } from '@ember/object';
+/* eslint-disable ember/no-computed-properties-in-native-classes */
 import FlashMessage from 'ember-cli-flash/components/flash-message';
 
-export default FlashMessage.extend({
+export default class FlashMessageComponent extends FlashMessage {
   // override alertType to get Bulma specific prefix
-  //https://github.com/poteto/ember-cli-flash/blob/master/addon/components/flash-message.js#L35
-  alertType: computed('flash.type', {
-    get() {
-      const flashType = this.flash.type || '';
-      let prefix = 'is-';
-
-      return `${prefix}${flashType}`;
-    },
-  }),
-});
+  //https://github.com/poteto/ember-cli-flash/blob/master/addon/components/flash-message.js#L55
+  get alertType() {
+    const flashType = this.args.flash.type || '';
+    return `is-${flashType}`;
+  }
+}
```

Comment:
This needed to be updated after upgrading to the latest version since they converted to native class.